vip-helpers/vip-elasticsearch.php

```diff
@@ -469,6 +469,12 @@ function vip_elasticsearch_filter_ep_index_name( $index_name, $blog_id, $indexab
 	return $index_name;
 }
 
+function vip_elasticsearch_filter_ep_do_intercept_request( $request, $query, $args, $failures ) {
+	$request = vip_safe_wp_remote_request( $query['url'], null, 3, 1, 20, $args );
```

Comment:
Instead of passing `null` for the fallback, we need to pass a `WP_Error()`, otherwise we'll literally get `null` back when it fails due to how PHP's default function args work.

Reply:
This is done, in b90e0d2e